database/migrations/2021_12_11_133514_create_trainig_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateTrainigTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('trainig', function (Blueprint $table) {
+            $table->foreignId('clients_id')
```

Comment:
it is preferable to keep foreign key in singular as `client_id` and `work_type_id`

Reply:
it still should work with empty `constrained`

Reply:
Please apply for all places.

Reply:
fixed